evm/deploy-upgradable.js

```diff
@@ -150,6 +159,7 @@ async function deploy(options, chain) {
         if (!yes) {
             const anwser = readlineSync.question(`Perform an upgrade for ${chain.name}? ${chalk.green('(y/n)')} `);
             if (anwser !== 'y') return;
+            console.log('');
```

Comment:
Linter?

Reply:
just adding a newline so next log is more readable